### PR TITLE
fix(core): CSS for details panel

### PR DESCRIPTION
**Pull request by caseyhebebrand**
Two css fixes: 
1. The project header being full-width was being overwritten by custom styles. This moves the `width: 100%` to a more specific set of classes to ensure there are no issues.
2. Tweaking the height of the details panel so there are no scrolling issues.

### CODE REVIEW

File: app/scripts/modules/core/src/presentation/main.less [hunk 1]
```diff
@@ -289,9 +289,9 @@ html {
     position: fixed;
     margin-top: -20px;
     right: 0;
-    height: 100%;
+    height: 98%;
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/presentation/main.less:292]: this seems odd, why?

**Comment by christopherthielen** [comment]
what custom styles were overriding the width, and how were the selectors getting overridden?

**Comment by caseyhebebrand** [comment]
> what custom styles were overriding the width, and how were the selectors getting overridden?

There are generic classes in the FP css that were overriding the use of these classes in core. I also am going to make a supplementary PR that removes styling from the general classes so it can be maintained from one place.

File: app/scripts/modules/core/src/presentation/main.less (hunk 1 above)
Comment (caseyhebebrand) [reply app/scripts/modules/core/src/presentation/main.less:292]: I noticed that the width at 100% was pushing the drawer out of the screen height, which is why the scrolling issues were cropping up. This was more dramatic from a projects view or with a custom header.

File: app/scripts/modules/core/src/projects/project.less [hunk 2]
```diff
@@ -57,6 +58,12 @@
       margin-top: -10px;
     }
   }
+
+  .insight {
+    .detail-content {
+      height: calc(~'100% - 54px');
```
Comment (gcomstock) [inline app/scripts/modules/core/src/projects/project.less:64]: Is it worth storing 54px as a variable that's shared between here and the header height?

Comment (caseyhebebrand) [reply app/scripts/modules/core/src/projects/project.less:64]: 👍

**Pull request merged**